slag/flag.go

```diff
@@ -0,0 +1,26 @@
+package slag
+
+import (
+	"flag"
+	"log/slog"
+	"strings"
+)
+
+type Level slog.Level
+
+func (l *Level) String() string { return slog.Level(*l).String() }
+func (l *Level) Set(s string) error {
+	var ll slog.Level
+	if err := ll.UnmarshalJSON([]byte(strings.ToUpper(s))); err != nil {
+		return err
+	}
+	*l = Level(ll)
```

Comment:
Playing around with this, I think this needs to be quoted to work? https://go.dev/play/p/nw0ZJxpFLSt

(worst case we can make this a switch)

Reply:
This should be `UnmarshalText` 🤦